tests/RelationsTest.php

```diff
@@ -104,4 +105,12 @@ public function testBelongsToManyReturnsManyToManyRelationship()
         /** @noinspection PhpParamsInspection */
         $this->assertInstanceOf(BelongsToMany::class, $relation);
     }
+
+    public function testBelongsToOneReturnsOneToOneRelationship()
+    {
+        $relation = (new Relations())->belongsToOne('test', TestModel::class);
+
+        /** @noinspection PhpParamsInspection */
```

Comment:
PHPStorm claims that this suppression is redundant. I don't even remember why it's there. Feel free to remove it.